NEWS

```diff
@@ -116,6 +116,10 @@ New Functionality
   ``make-archive-name`` script and using the new ``ZEEK_ARG_LOG_SUFFIX``
   environment variable.
 
+- A supervisor controlled Zeek cluster is now multi-logger aware. This avoids
+  loggers overwriting each others log files within a single log-queue directory.
```

Comment:
Total nit: 
```suggestion
  loggers overwriting each others' log files within a single log-queue directory.
```

Reply:
Hmm, I added a `'`. After a bit of googling I've put it before the "s" feel free to fixup.